### ISSUE TITLE
AbsoluteSum problem

### ISSUE THREAD

**Issue by leemcsorley** [kunzmi/managedCuda#70]
I think there's a problem in: 

void AbsoluteSum(CudaDeviceVariable<double> x, int incx, CudaDeviceVariable<double> result)

I'm getting an exception thrown no matter how I use it. I think it's calling the wrong native function:

 CudaBlasNativeMethods.cublasIdamax_v2 instead of the asum one?

Thanks.

**Comment by leemcsorley** [comment]
https://github.com/kunzmi/managedCuda/blob/e796f78f8ec72bfea1dfe1974f1aaa5d758ef020/CudaBlas/CudaBlasHandler.cs#L1584

**Comment by kunzmi** [comment]
Yes you're right, this is somehow wrong. I'll fix it latest with the upcoming 10.1 update!
